app/components/request_form/request_form.rb

```diff
@@ -2,20 +2,17 @@
 
 module RequestForm
   class RequestForm < ApplicationComponent
-    def initialize(organization:, request:)
+    def initialize(organization:, request:, available_tags:)
```

Comment:
Well, it has already been here. Why aren't we deducting the organization from the request itself?

Reply:
Good point